### ISSUE TITLE
Are there any known Kontakt library issues? 

### ISSUE THREAD

**Issue by Phfraiengngchkxzcs** [jtackaberry/reaticulate#81]
I've been using Reaticulate for about 6 months and it's all working smoothly except for a couple things and I was curious if these issues might be known to others. the most bizarre is with the 8dio Agitato Grandiose strings. When i'm programing my strings, they seem to work as normal. my legato articulation maps to legato. legato gentle to legato gentle, legato detache to legato detache. however, when i render stems the articulations shift by 1. legato becomes legato gentle, legato gentle becomes legato detache, etc... and it seems to stay that way. i then have to remap the articulations and re-render. then it's fine. but it's strange and kind of time consuming. 
Similarly, i can't seem to get the Metropolis Ark Finckenstein and Wolfenstein strings to map correctly via reaticulate. no matter what articulation I enter (and i have about 200 other libraries mapped and working correctly) only 1 articulation is ever triggered on the actual Kontakt instrument. i have tried shifting the keymap so different notes would be used to trigger, but the same result. i can only trigger the crescendo articulation, regardless of what trigger is being sent by reaticulate.

**Comment by jtackaberry** [comment]
> however, when i render stems the articulations shift by 1. legato becomes legato gentle, legato gentle becomes legato detache, etc... and it seems to stay that way. i then have to remap the articulations and re-render. then it's fine. but it's strange and kind of time consuming.

Is this a new issue with 0.4.x (which I assume you're using, you didn't say) or has it always happened?

I'm curious what happens if you use some of Reaper's other render options in the Render screen.  Does this still occur if you set it to 1x Offline?  What about Online Render?  Does it occur if you bypass Reaticulate and include the keyswitches directly in your MIDI item?

> but the moment i hit play or render, the pan goes 100% to the right. i don't know if it's the articulation that's doing that or what. i used "show all used cc data" but i don't see anything out of the ordinary.

Is this Kontakt's pan or the pan of the track in Reaper?  Assuming the former, it sounds like there is some CC mapped to the pan slider and Reaper is chasing this CC.  Perhaps if you're able to create a simple test project and share it I could dig in a bit deeper.

> Similarly, i can't seem to get the Metropolis Ark Finckenstein and Wolfenstein strings to map correctly via reaticulate. no matter what articulation I enter (and i have about 200 other libraries mapped and working correctly) only 1 articulation is ever triggered on the actual Kontakt instrument. 
I don't have this library I'm afraid.  Are you able to activate articulations directly by sending the appropriate keyswitches?

**Comment by Phfraiengngchkxzcs** [comment]
Hi Jason, 

Thanks for getting back to me on this! I do not believe this is a new issue. As I recall this is something that I’ve seen from the beginning and have just kind of avoided using this string bank for a while. 

I’m not actually using the render function. I didn’t say that clearly. I’m running a cycle action designed to go through my folder tracks that are setup for a single instance of Vienna (eg., my string section which is going to a Vienna plugin which houses 10 or so instances of Kontakt). So the basic process is that it solos and prints each midi clip in succession and then mutes the tracks and collapses the folders, disabling the plugins on the tracks in the process. 


However, with this particular group of instruments it doesn’t function correctly. And it seems to actually jump back-and-forth so I’m wondering if it’s something with these particular instruments. I did some testing, and it was really bizarre. I went through and changed the program data by 1 to trigger it correctly, and it seemed to switch back, so that I was now triggering one PC above the correct patch. It didn’t require the print process at all, it just seems to get confused 1/2 way through. 

The pan is on the Kontakt instrument and not in reaper. I will dig deeper on the CC data. I couldn’t find anything mapped to it but will look and see if there’s some data hidden somewhere. I’d be happy to make the test project but without the library I don’t think you’d be able to get any info. 

In regard to the Finckenstein and Wolfenstein strings, yes I can trigger those correct via key switch. So it’s not a huge deal. But I’ve just been trying to map everything so that the PC data will translate from library to library and I don’t have to remap stuff to print string layers, etc.. those are kind of specialty patches however and I can work around that. Just curious if you’d heard anything from other users. 

Thanks! I’ll keep messing with it and report any further findings. I think everything else is rockin’. Like I said, I use a lot of libraries and have them mapped to Reaticulate and then run them with Lemur. 
The only issue I’ve seen, and I think this is a Lemur issue, is that if I change tracks and quickly hit play, or change tracks while playing, I get a pretty big stutter, and that is when Lemur is changing the screen to catch up to the new Reaticulate map. Could just be that that software is a little slow in processing all that data (?) but again, curious if you’ve heard anybody asking about that. 

Thanks!


Ryan Franks
(323) 333-6778
 <http://rsbrigade.com/>TheRSBrigade <https://www.rsbrigade.com/>
IMDb <https://www.imdb.com/name/nm1766391/?ref_=fn_al_nm_1>

**Comment by jtackaberry** [comment]
Nope, you're the first I've heard reporting these issues.  Isn't that always the way. :)

I don't really have any ideas I'm afraid based on what you wrote, mainly just because of the underlying complexity and I suspect there are many relevant details that can't easily be described.  On the first issue, if you're able to condense the problem down into a test project I'm happy to take a closer look.

**Comment by Phfraiengngchkxzcs** [comment]
Thanks, as always, Jason. I appreciate the help. I will keep working with it and see if I can make it consistently happen and/or not happen in certain contexts. There are a lot of variables at play and I’ll see if I can make a test project for it. 
Thanks! 

Ryan Franks
(323) 333-6778
 <http://rsbrigade.com/>TheRSBrigade <https://www.rsbrigade.com/>
IMDb <https://www.imdb.com/name/nm1766391/?ref_=fn_al_nm_1>

**Comment by jtackaberry** [comment]
I'm wondering @Phfraiengngchkxzcs if this issue is still affecting you, and if it is, do you have any more insight as to whether it's a defect with Reaticulate or something related to Kontakt or the VIs mentioned?

**Comment by Phfraiengngchkxzcs** [comment]
> I'm wondering @Phfraiengngchkxzcs if this issue is still affecting you, and if it is, do you have any more insight as to whether it's a defect with Reaticulate or something related to Kontakt or the VIs mentioned?

Thanks @jtackaberry i can't say that i have further insight, maybe a couple more details. The problem so far as i can tell, has held consistent after now 6 months of working this way. 8dio's Grandioso string libraries do not respond correctly to Reaticulate, seeming to jump to 1 articulation above what is chosen during bounce and sometimes playback. LA scoring strings are getting an odd glitch where only the Trill and Tremelo patches jump to hard right panning when triggered and must be accompanied with CC pan data to correct. and the Metropolis Ark Finckenstein and Wolfenstein strings will only trigger the crescendo patch no matter what articulation is sent to them. 

I messed with these a lot when i was first setting them up, but to be honest I have just worked around them as they are not heavy-use libraries for me. However, in the next few days i'd be happy to make some test sessions and send them. I'm not totally sure on what i'd need to send you though. Would you want me to generate these issues and bounce them to audio and include the midi items with the PC data in them? I can send you the session. do you have VEPro 6? i am running Reaper with VEP6 which houses my kontakt instances. Let me know what i could get you that would be helpful and I'll do it. Thanks!
